host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_nvcc_xlf.cmake

```diff
@@ -108,6 +108,8 @@ set(CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "")
 
 set(AXOM_CUDA_ARCH "sm_70" CACHE PATH "")
 
+set(AXOM_USE_CUB ON CACHE BOOL "")
```

Comment:
Also -- our ``CMake`` configuration options are all of the form ``AXOM_ENABLE_*``, while their corresponding C++ defines are of the form ``AXOM_USE_*``.  

Please update this for consistency.

Reply:
Fixed -- Also updated the `AXOM_USE_MPI3` variable accordingly.